shared/data/fingerprint-protection.js

```diff
@@ -73,30 +73,20 @@
                 'targetValue': 8
             }
         },
-        'storage': {
-            'webkitTemporaryStorage': {
-                'object': 'navigator',
-                'origValue': navigator.webkitTemporaryStorage,
-                'targetValue': undefined
-            },
-            'webkitPersistentStorage': {
```

Comment:
I'm wondering why are we changing `webkitTemporaryStorage`? and why are we not overwriting `webkitPersistentStorage ` anymore? Description doesn't mention this.

Reply:
Removing temp storage was causing breakage on some sites that use it (it's a Chrome only feature, and that site would tell users that it only works on chrome...)

On further reviewing during troubleshooting, persistent storage always returns 0, and prompts for permission to change it, so removing it seemed a safer option since I don't think it can be used to FP without permission.

Reply:
Ah, good find re webkitPersistentStorage, I didn't know that.